docs/sources/tempo/configuration/_index.md

```diff
@@ -1572,3 +1475,159 @@ This value is available in the the [tempo-distributed](https://github.com/grafan
 # -- If true, Tempo will report anonymous usage data about the shape of a deployment to Grafana Labs
 reportingEnabled: true
 ```
+
+## Cache
+
+Use this block to configure caches available throughout the application. Multiple caches can be created and assigned roles
+which determine how they are used by Tempo.
+
+```yaml
+cache:
+    # Background cache configuration. Requires having a cache configured. These settings apply
+    # to all configured caches.
+    background:
+
+        # at what concurrency to write back to cache. Default is 10.
+        [writeback_goroutines: <int>]
+
+        # how many key batches to buffer for background write-back. Default is 10000.
+        [writeback_buffer: <int>]
+
+
+    caches:
+
+        # Roles determine how this cache is used in Tempo. Roles must be unique across all caches and 
+        # every cache must have at least one role. Allowed values:
+        # bloom              - Bloom filters for trace id lookup.
+	      # parquet-footer     - Parquet footer values. Useful for search and trace by id lookup.
+	      # parquet-column-idx - Parquet column index values. Useful for search and trace by id lookup.
+	      # parquet-offset-idx - Parquet offset index values. Useful for search and trace by id lookup.
+
+    -   roles:
+        - <role1>
+        - <role2>
+
+        # Memcached caching configuration block
+        memcached:
+
+            # hostname for memcached service to use. If empty and if addresses is unset, no memcached will be used.
+            # Example: "host: memcached"
+            [host: <string>]
+
+            # Optional
+            # SRV service used to discover memcache servers. (default: memcached)
+            # Example: "service: memcached-client"
+            [service: <string>]
+
+            # Optional
+            # comma separated addresses list in DNS Service Discovery format. Refer - https://cortexmetrics.io/docs/configuration/arguments/#dns-service-discovery.
+            # (default: "")
+            # Example: "addresses: memcached"
+            [addresses: <comma separated strings>]
+
+            # Optional
+            # Maximum time to wait before giving up on memcached requests.
+            # (default: 100ms)
+            [timeout: <duration>]
+
+            # Optional
+            # Maximum number of idle connections in pool.
+            # (default: 16)
+            [max_idle_conns: <int>]
+
+            # Optional
+            # period with which to poll DNS for memcache servers.
```

Comment:
```suggestion
            # Period with which to poll DNS for memcache servers.
```

Reply:
Paging down through the file it seems inconsistent. I think I'd prefer starting with a capital. WDYT?

I'd be glad to update all of the lines in the caching block I just added, but I don't think I should cleanup the whole file in this PR (would add a lot of noise).

Reply:
I think using the sentence case would be correct, and would be great if you want to make your section consistent. I think we have a doc issue open to make the doc structure for the configuration page the same. We can tackle the consistency across the entire document in that issue.